docs/iis-instrumentation.md

```diff
@@ -0,0 +1,27 @@
+# Instrument an ASP.NET application deployed on IIS
+
+## Instrument an ASP.NET 4.x application
+
+You can instrumental all ASP.NET 4.x application deployed to IIS
+by setting the required environment variables for
+`W3SVC` and `WAS` Windows Services as described in [windows-service-instrumentation.md](windows-service-instrumentation.md).
+
+> Unfortunately it is not possible to set distinct environment variables
```

Comment:
There is an option to set environment variables at the application pool level. They could use this feature as a workaround for ASP.NET app.

Ref: https://docs.microsoft.com/en-us/iis/configuration/system.applicationhost/applicationpools/add/environmentvariables/

Reply:
Thanks. That is great.